### DISCUSSION TITLE
Allow "interactive messages" in campaigns

### DISCUSSION THREAD

**Discussion by jalberto** [chatwoot/chatwoot#8469] [Feature Requests]
**Describe the solution you'd like**
Some enhancements I would love to see:

- To use interactive messages as part of campaigns
  - this will add much more flexibility in how we can interact and communicate with users
- More flexibility in "where" to display "campaigns interactive messages"
  - as a banner in top/bottom
  - as a modal in the center
  - this would be ideal for important notifications/warnings/etc
- Custom embedded forms
  - to be able to send form data to an external webhook

**Additional context**

https://www.chatwoot.com/docs/product/others/interactive-messages/

**Comment by jacekmichalski** [comment]
+1 for interactive messages in campaign message.

I was thinking about using options interactive message to pick up the dialogflow chatbot intent. It would be so helpful.

E.g. 
On e-commerce site I have an ongoing campaig with interactive message "Do you want to buy this product cheaper? - yes/no" and the dialog flow could automate coupon codes creation based on google cloud functions and intents.

Could you please take a look at it?

**Comment by pranavrajs** [comment]
I'm moving these to discussions as we will not be prioritizing these immediately.
